sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-storage-resource/README.adoc

```diff
@@ -0,0 +1,115 @@
+= Spring Cloud Azure Storage Starter Sample
```

Comment:
We already have the sample in our samples, https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-storage-resource, so I suggest we remove this one.

Reply:
> @yevster I've pushed a commit to your branch to solve conflicts. Please remember to pull before push if you want to add more commits.

@saragluna, I had in-flight changes fixing the event hub binders sample. Now, I see that sample, along with several others have been removed. May I ask why?